server/utils/convert-date.util.ts

```diff
@@ -1,5 +1,19 @@
 import { HDate } from "@hebcal/core";
+import { SpecificDateType } from "../../lib/types/minyan.type";
 
 export const convertHDateToDate = (hDate: HDate): Date => {
   return new Date(hDate.getFullYear(), hDate.getMonth() - 1, hDate.getDate());
 };
+
+export const convertToHebrewDayAndMonth = (date: Date): SpecificDateType => {
+  const hDate = new HDate(date);
+  return {
+    date,
+    hebrewMonth: hDate.getMonthName(),
+    hebrewDayMonth: hDate.getDate().toString(),
+  };
+};
+export const getHebrewMonthNumber = (monthName: string): number => {
+  const months = ["Nisan", "Iyar", "Sivan", "Tammuz", "Av", "Elul", "Tishrei", "Cheshvan", "Kislev", "Tevet", "Shevat", "Adar", "Adar II"];
+  return months.indexOf(monthName) + 1;
+};
```

Comment:
You can use the HDate class from the @hebcal/core library to get the Hebrew month number more directly. Here's an updated version of the getHebrewMonthNumber function using HDate:


```suggestion
export const getHebrewMonthNumber = (monthName: string): number => {
  const hDate = new HDate();
  const months = hDate.getMonthNames();
  return months.indexOf(monthName) + 1;
};
```

This approach leverages the HDate class to retrieve the month names directly from the library, ensuring consistency with the Hebrew calendar.